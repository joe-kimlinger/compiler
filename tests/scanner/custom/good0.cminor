// identifiers
// length = 255 + null
_12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234
_can_start_with_underscore
c090393023s_ds
_cds9112
p
_
ALL_CAPS
