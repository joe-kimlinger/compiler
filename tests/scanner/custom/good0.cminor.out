COMMENT
COMMENT
IDENTIFIER _12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234
IDENTIFIER _can_start_with_underscore
IDENTIFIER c090393023s_ds
IDENTIFIER _cds9112
IDENTIFIER p
IDENTIFIER _
IDENTIFIER ALL_CAPS
